source/declarations.tex

```diff
@@ -1512,7 +1512,9 @@
 \indextext{name!elaborated!\idxcode{enum}}%
 If an \grammarterm{elaborated-type-specifier} is the sole constituent of a
 declaration, the declaration is ill-formed unless it is an explicit
-specialization\iref{temp.expl.spec}, an explicit
+specialization\iref{temp.expl.spec},
+a partial specialization\iref{temp.spec.partial},
+an explicit
 instantiation\iref{temp.explicit} or it has one of the following
```

Comment:
I think that is an "or" missing in the preexisting text here; suggestion:

> If an elaborated-type-specifier is the sole constituent of a declaration, the declaration is ill-formed unless
> 
> - it is an explicit specialization (13.9.4 [temp.expl.spec]), a partial specialization (13.7.6 [temp.spec.partial]),<ins> or</ins> an explicit instantiation (13.9.3 [temp.explicit]); or
> - it has one of the following forms:

Reply:
Absolutely! added in a separate commit.